addons/website_hr_recruitment/views/website_hr_recruitment_templates.xml

```diff
@@ -3,81 +3,126 @@
-                                    />
-                                    <div class="o_job_infos mt16">
-                                        <span t-field="job.address_id" t-options='{
-                                            "widget": "contact",
-                                            "fields": ["address"],
-                                            "no_tag_br": True
-                                            }'/>
-                                        <div t-if="job.contract_type_id">
-                                            <i class="fa fa-fw fa-suitcase" title="Employment type" role="img" aria-label="Employment type"/><span t-field="job.contract_type_id"/>
-                                        </div>
-                                        <div>
-                                            <i class="fa fa-fw fa-clock-o" title="Publication date" role="img" aria-label="Publication date"/><span t-field="job.write_date"/>
-                                        </div>
-                                    </div>
+                            <div class="alert alert-info p-3 mx-lg-4" groups="hr_recruitment.group_hr_recruitment_manager">
+                                <div class="text-center text-muted">
+                                    <h5>Create new job pages from the <strong>+ <i>New</i></strong> top-right button.</h5>
                                 </div>
                             </div>
-                        </a>
+                            <div class="col-lg-12 text-center pt24">
+                                <h5 class="fw-light text-muted">There are currently no open job opportunities,</h5>
+                                <h5 class="fw-light text-muted pb24">but feel free to <span class="fw-bold">contact us</span> for a spontaneous application.</h5>
+                                <img class="img-fluid o_we_custom_image pb24" src="/website_hr_recruitment/static/src/img/job_not_found.svg" style="width: 50% !important;" alt="Job not found"/>
+                            </div>
+                        </div>
+                    </t>
+                    <div t-foreach="jobs" t-as="job" class="col-lg mb32">
+                        <div t-attf-class="card #{not job.website_published and 'o_jobs_unpublished'}">
+                            <a t-attf-href="/jobs/detail/#{ slug(job) }" t-attf-class="text-decoration-none text-reset">
+                                <div class="card-body p-4">
+                                    <t t-call="website_hr_recruitment.job_index_description"/>
+                                </div>
+                            </a>
+                        </div>
                     </div>
                 </div>
             </div>
         </div>
+    </div>
+</template>
+
+<!-- Job Index Description -->
+<template id="job_index_description" name="Job Index Description">
```

Comment:
I'll add a commit which only extracts this in a dedicated template first. Again: please think of the reviewer.